programs/programs/policyengine/calculators/dependencies/member.py

```diff
@@ -449,3 +449,27 @@ class HeadStart(Member):
 
 class EarlyHeadStart(Member):
     field = "early_head_start"
+
+
+class IsFppMedicaidEligibleDependency(Member):
+    """
+    Dependency for is_medicaid_eligible used by IL Family Planning Program.
+
+    This checks if the member is enrolled in full-time Medicaid or other
+    full coverage state medical program, which makes them ineligible for FPP.
+    """
+
+    field = "is_medicaid_eligible"
+    dependencies = ("insurance",)
+
+    def value(self):
+        # Check if member has Medicaid or family_planning insurance coverage
+        if hasattr(self.member, "insurance"):
+            return self.member.insurance.has_insurance_types(("medicaid", "family_planning"), strict=False)
+        return False
```

Comment:
Remove this dependency. PE can calculate this. We'll also reach out to Ziming about using `receives_medicaid` instead of `is_medicaid_eligible` for this eligibility calculation